File: 217/solution.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        numcnt = {}
        for i in nums:
            if i in numcnt:
                return True
            else:
                numcnt[i] = i
        return False

File: 217/solution.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_map <int, int> map;
        for(auto i:nums){
            if(map.find(i) == map.end())
                map[i] = i;
            else
                return true;
        }
        return false;
    }
};
